### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Hyper-Node**
# MaRDI Pull Request
Debugging with xdebug is now possible. Readme shows how to set up debugging in docker-compose override. 

 
**Instructions for PR review**:
- [x] Conceptual Review (Logic etc...) 
- [ ] Code Review (Review your implementation) 
- [ ] Checkout (Test changes locally) 

**Checklist for this PR**: 
- [x] [Reviewers and Assignee specified.](https://docs.github.com/en/issues/tracking-your-work-with-issues/assigning-issues-and-pull-requests-to-other-github-users)
- [x] [All related issues are linked.](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,13 +131,18 @@ version: '3.4'
...
       XDEBUG_CONFIG: "remote_host=host.docker.internal"
+    expose:
+      - 9000
     volumes:
-     - ~/git/mediawiki/MathSearch:/var/www/html/extensions/MathSearch
+      - ./extensions-dev/<extension_to_debug>:/var/www/html/extensions/<extension_to_debug>
+      - ./debugging/php.ini:/usr/local/etc/php/php.ini
 ```
-Here `~/git/mediawiki/MathSearch` is the path of your local development checkout of the extension, you modify.
+Here `~/git/mediawiki/<extension_to_debug>` is the path of your local development checkout of the extension, you modify.
```
Comment (dajuno) [inline README.md:143]: path should be probably `./extensions-dev/...`

Comment (Hyper-Node) [reply README.md:143]: zend installation is based on wikibase container setup, so it's non specfic.

Comment (dajuno) [reply README.md]: ok. in that case shouldnt debugging/php.ini be in docker-wikibase?

**Review by dajuno — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Hyper-Node) [reply README.md]: it is loaded by docker compose, and can be customized (stuff like log-level ) this is easier to set in portal-compose

**Pull request merged**